src/components/CaseItemsCombined.js

```
@@ -101,12 +97,12 @@ const JobItemsCombined = () => {
                </tr>
                </thead>
                <tbody>
-               {jobItems.map((item) => (
+               {caseItems.map((item) => (
                   <tr>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)